### ISSUE TITLE
Force cache resolve ?

### ISSUE THREAD

**Issue by lucFM** [liip/LiipImagineBundle#966]
| Q | A
| --- | ---
| Bug Report? | no
| Feature Request? | yes
| BC Break Report? | no
| RFC? | no
| Imagine Bundle Version | 1.8.0

Hi, is there a way to force cache resolve ? 
I would like to re generate all my thumbnails without remove them.

Thanks

**Comment by robfrawley** [comment]
I actually needed exactly the same thing recently and had amended the console command to accomplish it. I turned my temporary code into a proper pull request in #967 --- seeh ow that works for you with the added option `--force` (which will force the resolved image to be regenerated, regardless of cache). If you want to do this in a batch fashion, you can do something like:

```bash
for f in "$(ls path/to/images/*.jpg)"; do
  bin/console liip:imagine:cache:resolve --filter=my_filter_name "${f}"
done
```

**Comment by robfrawley** [comment]
An option to force resolution (regardless of the cache state) has been added to the console command (with the addition of the `--force` CLI option) as #967 for the `1.x` branch (merged) and #981 for the `2.x` branch (in review).

**Comment by lucFM** [comment]
awesome !
